### PR TITLE
Feature: add vertex_ai/claude-3-opus@20240229 to the model list

### PR DESCRIPTION
**Pull request by Manouchehri**

### CODE REVIEW

**Comment by Manouchehri** [comment]
Confirmed to work in `us-east5`.

image

File: litellm/model_prices_and_context_window_backup.json [hunk 1]
```diff
@@ -1094,6 +1094,14 @@
         "litellm_provider": "vertex_ai-anthropic_models",
         "mode": "chat"
     },
+    "vertex_ai/claude-3-opus@20240229": {
+        "max_tokens": 200000,
+        "max_output_tokens": 4096,
+        "input_cost_per_token": 0.000015,
+        "output_cost_per_token": 0.000075,
+        "litellm_provider": "vertex_ai-anthropic_models",
+        "mode": "chat"
```
Comment (ghost) [inline litellm/model_prices_and_context_window_backup.json:1103]: @Manouchehri do you mind adding source, so we know where the pricing is from? 

For vertex i always find it hard to navigate their docs and grab the pricing

Comment (Manouchehri) [reply litellm/model_prices_and_context_window_backup.json:1103]: https://cloud.google.com/vertex-ai/generative-ai/pricing#partner-models
image

**Comment by Manouchehri** [comment]
Duplicate of ab1e26ad89d7f29e8ae47b1187331bb620cc168a.

**Pull request closed without merging**